client/components/deposits-overview/index.tsx

```diff
@@ -73,6 +74,18 @@ const DepositsOverview: React.FC = () => {
 		account?.deposits_schedule?.interval
 	);
 
+	const navigateToDepositsHistory = () => {
+		recordEvent( 'wcpay_overview_deposits_view_history_click' );
+
+		const history = getHistory();
+		history.push(
```

Comment:
⛏️🤏 tiny nitpick: is it better or worse to make this more concise?
```suggestion
		getHistory().push(
```

Reply:
I had the same thought, and then I went ahead with what is the most common way to use it, but I think for this case, we can lean either way.